admin_ui_support/src/ReadOnlyTableEntityStorage.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Drupal\admin_ui_support;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Entity\Sql\SqlContentEntityStorage;
+
+/**
+ * SQL Entity storage for entity types that wrap readonly tables.
+ */
+class ReadOnlyTableEntityStorage extends SqlContentEntityStorage {
```

Comment:
I'd expected to override ```\Drupal\Core\Entity\Sql\SqlContentEntityStorage::save``` to be overridden to ensure nothing can  ever save it

Reply:
I added this and throwing a logic exception if save() is called.

Also removed the need for WatchdogStorage class because it was only needed to handle watchdog table not having a UUID field. Move this logic to ReadOnlyTableEntityStorage